src/api/spec/features/webui/maintenance_workflow_spec.rb

```diff
@@ -71,7 +71,10 @@
     fill_in('reason', with: 'really? ok')
 
     click_button('Accept request')
-    expect(page).to have_css('#overview h3', text: "Request #{bs_request.number} accepted")
+    # Looks like accepting the request takes some time, so we allow it to take a bit more than usual
```

Comment:
Isn't this a page load? What does this have to do with `Capybara.default_max_wait_time`?

Reply:
When using the `have_*` matchers (and the finders also), Capybara waits some time before saying "I didn't find that thing you were expecting" and failing with an error. That is the `Capybara.default_max_wait_time` (that we have set to 6).

See https://github.com/teamcapybara/capybara?tab=readme-ov-file#asynchronous-javascript-ajax-and-friends (bear in mind that the waiting happens even if there is no AJAX involved)

Reply:
Which explicitly says it's about asynchronous javascript, but feel free to ignore me :)